ng-dev/pr/merge/defaults/labels.ts

```diff
@@ -57,8 +64,8 @@ export async function getDefaultTargetLabelConfiguration(
     {
       pattern: 'target: minor',
       // Changes labeled with `target: minor` are merged most commonly into the next branch
-      // (i.e. `master`). In rare cases of an exceptional minor version while being already
-      // on a major release train, this would need to be overridden manually.
+      // (i.e. `main`/`master`). In rare cases of an exceptional minor version while being
```

Comment:
```suggestion
      // (i.e. `main`). In rare cases of an exceptional minor version while being
```

Since we are just providing an example, lets use the preferred branch name as the example